frontend/scss/globals/_g-header.scss

```diff
@@ -122,9 +130,26 @@
 
   [aria-expanded="false"] + .details {
     visibility: collapse;
+    opacity: 0;
+  }
+  [aria-expanded="false"].collapsing + .details {
+    animation: .3s fade-out;
+    transition: opacity .3s linear, background-color .3s linear;
+    @keyframes fade-out {
+      from {
+        opacity: 1;
+        visibility: visible;
+      }
+      to {
+        opacity: 0;
+        visibility: collapse;
+      }
+    }
```

Comment:
This `fade-out` `animation` keyframe is necessary because you can't use `visibility` in transitions as it is not a scalar value.